<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DPythonConsole</class>
 <widget class="QWidget" name="DPythonConsole">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>819</width>
    <height>336</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Python Console</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*
{
	color: rgb(234, 234, 234);
	background-color: rgb(51, 51, 55);
}

QPushButton
{
	border-radius: 3px;
}

.QLineEdit
{
	font-weight: bold;
	border-width: none;
	color: rgb(170, 85, 255);
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>3</number>
       </property>
       <property name="rightMargin">
        <number>10</number>
       </property>
       <item>
        <widget class="QLabel" name="mCommandLabel">
         <property name="font">
          <font>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>    Command &gt;&gt;&gt; </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="mLineEditCommand">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>28</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="mBtnRun">
         <property name="minimumSize">
          <size>
           <width>48</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>48</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>Run</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="mBtnPrevious">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="toolTip">
          <string>Previous Cmd</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../images/images.qrc">
           <normaloff>:/images/themes/default/mActionArrowUp.svg</normaloff>:/images/themes/default/mActionArrowUp.svg</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="mBtnNext">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="toolTip">
          <string>Next Cmd</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../images/images.qrc">
           <normaloff>:/images/themes/default/mActionArrowDown.svg</normaloff>:/images/themes/default/mActionArrowDown.svg</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="mBtnClearOutput">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="toolTip">
          <string>Clear Output</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../images/images.qrc">
           <normaloff>:/images/themes/default/console/iconClearConsole.svg</normaloff>:/images/themes/default/console/iconClearConsole.svg</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTextEdit" name="mTextEditOutput">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../images/images.qrc"/>
 </resources>
 <connections/>
</ui>
